Apollonius_graph_2/include/CGAL/Parabola_2.h

```diff
@@ -59,7 +59,7 @@ class Parabola_2
 #else
   static const FT& STEP()
   {
-    static FT step_(2);
+    static const FT step_(2);
```

Comment:
I removed the function and now have a `const FT STEP(2)` in the code. After all it is for drawing.